data-reconciliation-app/docs/adr/03-reconciliation-logic.md

```diff
@@ -11,37 +11,37 @@ Once data is ingested into the app, Members/Users will query for the data to be
+- When a user requests for Reconciliation Report, logic on the current dataset stored in the network is executed for this user.
 - Result will not present other Users' information on a specific record, but just a comparison with them.
 
 ## When Reconciliation Happens
 
-This logic will be embedded on the reporting API, and anytime a member asks for their data to be reconciled (ask for a report), the application will do the necessary calculations based on the data this member has entered.
+This logic will be embedded in the reporting API, and anytime a member asks for their data to be reconciled (asks for a report), the application will do the necessary calculations based on the data this member has ingested.
 
-With this approach, Data Ingestion will be detached from any rules related to Reporting/Reconciliation.
+With this approach, Data Ingestion will be decoupled from any rules related to Reporting/Reconciliation.
 
-This will be done by querying the KV Store for the keys a User has submitted data for. Data related to keys that are unknown to this User will not be present in the final report, and not considered for reconciliation. Therefore the reconciled data they get is only related to their submitted information.
+This will be done by querying the KV Store for the keys a user has submitted data for. Data related to keys that are unknown to this user will not be present in the final report and will not be considered for reconciliation. Therefore, the reconciled data they get is only related to their submitted information.
 
 ## Scenarios
 
 ### Querying for a Specific Record
 
-Users may want to reconcile a specific record. This will be done by directly querying the record in the KV Storage, and comparing the value submitted by the User against others'.
+Users may want to reconcile a specific record. This will be accomplished by directly querying the record in the KV Storage and comparing the user's value to others.
 
-Checks will be done if User has submitted this specific record.
+Checks will be done if the user has submitted this specific record.
 
 ### Querying for all Data
 
-Users may want to reconcile all data they submitted. This will be done by scanning the KV Storage, identifying the records this User has submitted and the comparison logic will be the same used for a single record.
+Users may want to reconcile all the data they submitted. This will be accomplished by scanning the KV Storage, identifying the records this User has submitted, and applying the same comparison logic as for a specific record.
 
 ## Rules
 
 **Current User:** The User requesting a reconciled report.
 
-Each record will have all members' opinions for it. So for each record the reconciliation will be done by comparing the opinion of the current User against the other.
+Each record will have all members' opinions on it. So, for each record, the reconciliation will be performed by comparing the current user's opinion to others'.
```

Comment:
In all previous points, we use `user` and `key`/`value`. Now, this is mentioning `member`, `record` and `opinion`. Is it consistent?

Reply:
Have updated in most of the places to user and key/value.